lib/helper.js

```diff
@@ -44,7 +45,7 @@ var validationType = function(value, validation, key){
       return {value: value};
     }else{
       if(types[validation].type) validation = types[validation].type;
-      return {error: key + ': ' + value + ', has to be ' + validation + ' type.', param: key};
+      return {error: key + ': ' + escapeHtml(value) + ', has to be ' + validation + ' type.', param: key};
```

Comment:
If we are responding with JSON, then this isn't the right place to do HTML escaping. Can we just escape the value when we output it client side?